tests/console/yast2_i.pm

```diff
@@ -81,20 +81,13 @@ sub run() {
         send_key 'alt-o';
     }
 
-    # Whether summary is shown depends on PKGMGR_ACTION_AT_EXIT in /etc/sysconfig/yast2
```

Comment:
do you see the comment you just removed? If updating the system, the sysconfig will tell yast not to show a summary. *That* is the reason why we have this variable, so we can set this in upgrade cases. If it's set in the wrong test suites, we should fix that.